src/views/Login.vue

```diff
@@ -6,6 +6,7 @@ import UserServices from "../services/UserServices.js";
 
 const router = useRouter();
 const isCreateAccount = ref(false);
+let isLoading = ref(false);
```

Comment:
other than setting this to true or false, I don't see where this ref value is read and used to do anything